app/lib/project/UFOExportController.js

```diff
@@ -0,0 +1,82 @@
+define([
+    'metapolator/errors'
+  , 'metapolator/math/hobby'
```

Comment:
I think some of these dependencies are outdated (they where even before your rewrite). Could you please remove the unused dependencies? It's a good occasion.